src/app/pages/institution-list/institution-list.component.ts

```diff
@@ -0,0 +1,10 @@
+import { Component } from '@angular/core';
+import { InstitutionComponent } from '../institution/institution.component';
+
+@Component({
+  selector: 'app-institution-list',
+  templateUrl: './institution-list.component.html',
+  styleUrls: ['./institution-list.component.css']
+})
+export class InstitutionListComponent {
```

Comment:
Esta muy bien, lo único que cambiaría para matar dos pájaros de un tiro, es crear antes el modelo de Institution, fíjate que es una tarea que tenes asignada. Una vez creado ese modelo, creado el pe y mergeado a develop, mergeate la rama develop acá y create acá una lista de institutions, debería quedar así:
`institutions:Institution[]`
`ngOnInit(){
	this.institutions = [new Institution(datos de ejemplo), new Institution(mismos datos de ejemplo)]
}
`

Con esto ya adelantamos un montón para cuando vengan los datos de la API, buen laburo 👏